README.md

```diff
@@ -1,10 +1,18 @@
 # dat.GUI
-A lightweight graphical user interface for changing variables in JavaScript. 
+@MacroMan's fork with extra features
+
+A lightweight graphical user interface for changing variables in JavaScript.
 
 Get started with dat.GUI by reading the [tutorial](http://workshop.chromeexperiments.com/examples/gui)
 or the [API documentation](API.md).
 
 
+## MacroMan's extra features
+* hide/show methods on GUI instances
+* hide/show methods on Controllers
```

Comment:
It's fine if you'd like to add this in the Changelog, but I'm not sure these changes make sense to merge in the official repo right?

Reply:
I didn't realise my further commits would also go into this PR, sorry.
I will close this one and open another with just the suggested edits in a new PR.